pkg/controllers/interruption/controller.go

```diff
@@ -176,6 +178,11 @@ func (c *Controller) handleMessage(ctx context.Context, msg messages.Message) (e
 			}
 		}
 	}
+
+	if !isMatchedInstance {
+		log.FromContext(ctx).V(1).Info("no node claim found for instance", "EC2InstanceID", msg.EC2InstanceIDs())
```

Comment:
I'm not a fan of this. This is going to be massively noisy because the SQS queue is receiving all events from the account right now -- this means there are going to be a lot of these logs that are just going to cloud our regular logs

Reply:
I have run Karpenter Node for a long time and confirmed that the above logs are a little noisy.
Thank you for your advice.

I will close this PR if I can't come up with a more suitable resolution.